sql/updates/1.4.2--1.5.0.sql

```diff
@@ -20,7 +20,7 @@ CREATE TYPE _timescaledb_catalog.ts_interval AS (
     integer_interval        BIGINT
     );
 
--- q -- todo:: this is probably necessary if we keep the validation constraint in the table definition.
+-- q -- this is probably necessary if we keep the validation constraint in the table definition; do not remove without alternative
```

Comment:
What's the `q` here? Typo? I realize it was t
here before but probably makes sense to remove while you're cleaning this up anyway.

Actually, since this is a historical update script, then why not remove this line altogether? I am not even sure what it means and I doubt anyone will mess with this now.